src/qml/qgismobileapp.qml

```diff
@@ -375,7 +375,11 @@ ApplicationWindow {
             } else {
                 if ( currentRubberband && currentRubberband.model.vertexCount > 1 ) {
                   coordinateLocator.sourceLocation = mapCanvas.mapSettings.coordinateToScreen( currentRubberband.model.lastCoordinate )
-                } else {
+                }
+                else if ( geometryEditorsToolbar.editorRubberbandModel && geometryEditorsToolbar.editorRubberbandModel.vertexCount > 1 ) {
+                  coordinateLocator.sourceLocation = mapCanvas.mapSettings.coordinateToScreen( geometryEditorsToolbar.editorRubberbandModel.lastCoordinate )
+                }
+                else {
```

Comment:
Just a small comment on the styling. In general, in JS we write the curly braces on the same line as `} else {` . Is there a special QML reason to split them in two lines?

Reply:
It's true; the reason is that I'm accustomed to writing code in a format where I separate the `if` and `else` statements on different lines for better readability (I think QML's auto-formatting separates too).
but considering we need to maintain the project's coding format, I will make the necessary adjustments. Thank you very much for pointing this out!